DOCSP 27810 updating schema validation behavior (#589)
mongodb/docs-compass: source/validation.txt

Validation Rules
----------------

*Updated in version 1.35.1*

The validation editor supports :manual:`JSON Schema validation
</core/schema-validation/#json-schema>`, and validation with query
expressions using :manual:`query operators </reference/operator/query>`.

> When you hit the Update button, Compass updates to display a document

Reviewer: [nit] For UI instructions, we want to be more explicit in what the users have to do — it might be confusing for international audiences to read about "hitting" a button.

Also for the update button, we want to add the `:guilabel:` directive!
```suggestion
After you click the :guilabel:`Update` button, Compass updates to display a document
```

Author: Done! Thanks for your feedback!
